commands/root.go

```diff
@@ -15,6 +15,7 @@ import (
 func Run(args []string) error {
 	viper.SetEnvPrefix("mmctl")
 	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
+	viper.SetDefault("local-socket-path", "/var/tmp/mattermost_local.socket")
```

Comment:
What is the rationale behind choosing `/var/tmp/` as the directory for the socket? If I'm understanding correctly, the files in this directory are persisted between reboots, and we would like the socket to be created/deleted when the server starts/stops, right?

Reply:
The initial idea was to go with `/var/run` directory but it requires `sudo` privileges to create/delete files. Therefore I picked `/var/tmp`. We can go for /tmp` because we would want socket file to be removed after a reboot, if there is one.